app/models/smart_proxy.rb

```diff
@@ -90,6 +93,20 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
+  def version
```

Comment:
what do you think about simply storing the proxy version in the db and update it once we communicate with the proxy? (we could cut down the api calls in the about page and if using a header to report the version we dont need to make any extra call)